app/views/backups/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= render "apps/heading", app: @app, tab: "backups" %>
+<div class="row">
+  <div class="col-md-8">
+    <table class="table table-hover domains">
+      <thead>
+        <tr>
+          <th>Service</th>
+          <th>Backup date</th>
+          <th>Type</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <%= render @backups %>
+        <tr class="<%= "hidden" if @backups.any? %>" data-role="empty-table">
+          <td colspan="3" class="text-muted text-center">There are no backups</td>
+        </tr>
```

Comment:
Not sure what you mean? As in, you render the partial in the response? (That might be a nicer solution :D)